### DISCUSSION TITLE
Member request does not reveal WHO is requesting

### DISCUSSION THREAD

**Discussion by Juksefantomet** [community/community#102030] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

![image](https://github.com/community/community/assets/13452749/8811ccce-efc2-450c-89be-c83fb7758e95)

We have a larger organization with external consultants etc. and lately we've received a few emails regarding copilot request from members. It would be VERY useful if this actually displayed who was requesting this.

**Comment by vci-steven** [comment]
Same here. Without knowing who made the request then we cannot fulfill it.
